apis/tempo/v1alpha1/tempostack_types.go

```diff
@@ -10,6 +10,8 @@ import (
 
 // TempoStackSpec defines the desired state of TempoStack.
 type TempoStackSpec struct {
+	// TracingEnabled - TODO: find a better home.
+	TracingEnabled bool
```

Comment:
What would you recommend where that should live?

Reply:
I woulder if we should create a sub spec for internal observability. Is there any metrics related config we should set. Or perhaps sampling?

Reply:
y, could defently make sense. The thing is, what its used for. If it is only used for debugging purposes, I would hide it in the crd under `spec.internal.debugging`. Otherwise maybe something like `spec.internal.traceconfig` that keeps maybe things like `sampling` and `endpoint`.
Or a more "granular" option would be to place trace configs next to the component configurations. wdyt?